src/solvers/strings/string_refinement.h

```diff
@@ -27,6 +27,7 @@ Author: Alberto Griggio, alberto.griggio@gmail.com
 
 #include "string_constraint.h"
 #include "string_constraint_generator.h"
+#include "string_dependencies.h"
```

Comment:
:pick: Could forward declare `string_dependenciest` instead of including this file.